src/components/calcite-color-picker-hex-input/calcite-color-picker-hex-input.tsx

```diff
@@ -209,7 +209,9 @@ export class CalciteColorPickerHexInput {
 
     const withModifiers = altKey || ctrlKey || metaKey;
     const exceededHexLength = inputNode.value.length >= 6;
-    const focusedElement = el.shadowRoot.activeElement as HTMLInputElement;
+    const focusedElement = el.shadowRoot.activeElement.shadowRoot.querySelector(
```

Comment:
@jcfranco I tried to fix this, but I think this is the issue failing the tests. It seems since the input is no longer in light dom, the selection logic needs to change. Here's my failed attempt.